thrift/src/main/java/com/linecorp/armeria/server/thrift/ThriftDocServicePlugin.java

```diff
@@ -168,7 +224,8 @@ static ServiceInfo newServiceInfo(
             });
         }
 
-        return new ServiceInfo(name, functions, classes, endpoints, docStrings.get(name), exampleHeaders);
+        return new ServiceInfo(name, functions, classes, endpoints,
```

Comment:
I'm not sure though if it's a good idea to have `ClassInfo` and `ServiceInfo` (or `FieldInfo` and `FunctionInfo`) have the same docStringId format. Should we add some prefix? e.g. `classes/<className>/<fieldName>`